src/main/resources/l10n/JabRef_en.properties

```diff
@@ -2259,4 +2259,17 @@ Decryption_not_supported.=Decryption_not_supported.
 
 Cleared_'%0'_for_%1_entries=Cleared_'%0'_for_%1_entries
 Set_'%0'_to_'%1'_for_%2_entries=Set_'%0'_to_'%1'_for_%2_entries
-Toggled_'%0'_for_%1_entries=Toggled_'%0'_for_%1_entries
\ No newline at end of file
+Toggled_'%0'_for_%1_entries=Toggled_'%0'_for_%1_entries
+
+Check_for_updates=Check_for_updates
+Download_update=Download_update
+New_version_available=New_version_available
+Current_version=Current_version
+Remind_me_later=Remind_me_later
+Ignore_this_update=Ignore_this_update
+couldn't_connect_to_the_update_server=Couldn't connect to the update server.
+Please_try_again_later_and/or_check_your_network_connection=Please try again later and/or check your network connection
+To_see_what's_new_view_the_changelog=To see what's new view the changelog
+A_new_version_of_JabRef_has_been_released_and_can_be_updated_now=A new version of JabRef has been released and can be updated now
+JabRef_is_uptodate=JabRef is uptodate
```

Comment:
Underscores are required here. Also "uptodate" looks weird, does this actually exist? I would replace it with "up to date" or "up-to-date".